services/core-api/app/api/projects/project_summary/resources/project_summary_list.py

```diff
@@ -153,8 +153,7 @@ def post(self):
             project_summary.save()
             if project_summary.status_code == 'SUB':
                 if is_minespace_user():
-                    project_summary.send_project_summary_email_to_ministry(mine)
-                    project_summary.send_project_summary_email_to_proponent(mine)
+                    project_summary.send_project_summary_email(mine)
```

Comment:
Should not include the same change to the other resources handling PUT method? (core-api/app/api/projects/project_summary/resources/project_summary.py Line 151-152)

Reply:
great catch, I pushed up the code.